### PR TITLE
Topic recognition-Unet_model

### PR DESCRIPTION
**Pull request by putaojun-123**
There are 2 script files, one image file, and a README.md document.
Student ID: 45788167.

### CODE REVIEW

**Comment by putaojun-123** [comment]
[Uploading Readme.pdf…]()

**Comment by putaojun-123** [comment]
[Readme.pdf](https://github.com/shakes76/PatternFlow/files/5481566/Readme.pdf)

**Comment by putaojun-123** [comment]
This is the final pull request. I upload a Readme.pdf file to show the whole flow process with a dice_coef result of 98%. Please refer to that pdf. Thanks.

**Comment by SiyuLiu0329** [comment]
- Commit messages should be a lot more specific
- Code valid for Easy Difficulty
- Results are good 

Other Feedback
- Shakes commits should not have appeared on your branch
- Chose a more meaningful name for your repo, "test" can cause confusion in an open source repo
- Please group the `.png` files and put them in a sub-directory
- You should not need `from IPython.display import clear_output` in .py files

**Comment by SiyuLiu0329** [comment]
You should comment to clarify that you have submitted another PR

File: recognition/test/unet_model.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+def unet_model(output_channel=4, f=4):
```
Comment (shakes76) [inline recognition/test/unet_model.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
Duplicate.... closing

**Pull request closed without merging**